### ISSUE TITLE
[JENKINS-41488] Pipeline view automatically scrolls down

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/blueocean-plugin#3689]
---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gusgonnet72">gusgonnet72</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488" target="_blank">Pipeline view automatically scrolls down</a></i></summary>
<i><ul>
<li><b>status</b>: Reopened
<li><b>priority</b>: Minor
<li><b>component(s)</b>: blueocean-plugin
<li><b>label(s)</b>: pipeline, radiator
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 6
<li><b>watchers</b>: 16
<li><b>imported</b>: 2025-12-01
</ul></i>
</details>
<ul><li><i>environment</i>: <code>Jenkins 2.x</code></li></ul>
<details><summary><i>1 attachment</i></summary>

- [pipeline.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/35635/pipeline.png)
  > ![pipeline.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/35635/pipeline.png)
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/kzantow.png" title="kzantow's avatar" /> kzantow:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=285244&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-285244">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jdumay" class="user-hover" rel="jdumay">jdumay</a> I don't believe this is a duplicate of <a href="https://issues.jenkins.io/browse/JENKINS-38226" title="Radiator view based on favourited pipelines" class="issue-link" data-issue-key="JENKINS-38226"><del>JENKINS-38226</del></a> - I think what <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gusgonnet72" class="user-hover" rel="gusgonnet72">gusgonnet72</a> is asking for is a way to display the executing pipeline of an individual job.</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jdumay" class="user-hover" rel="jdumay">jdumay</a> I don't believe this is a duplicate of <a href="https://issues.jenkins.io/browse/JENKINS-38226" title="Radiator view based on favourited pipelines" class="issue-link" data-issue-key="JENKINS-38226"><del>JENKINS-38226</del></a> - I think what <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gusgonnet72" class="user-hover" rel="gusgonnet72">gusgonnet72</a> is asking for is a way to display the executing pipeline of an individual job.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/jamesdumay.png" title="jamesdumay's avatar" /> jamesdumay:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=285389&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-285389">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Apologies, I marked this as the wrong duplicate - its <a href="https://issues.jenkins.io/browse/JENKINS-41088" title="Show &quot;latest&quot; run results via fixed URL (dashboard like feature)" class="issue-link" data-issue-key="JENKINS-41088">JENKINS-41088</a>. We are going to build a great page for a Pipeline that shows you the latest visualization of your Pipeline which will be perfect for big screens!</p>
</pre>
</details>

</ul>
</details>

<p>Apologies, I marked this as the wrong duplicate - its <a href="https://issues.jenkins.io/browse/JENKINS-41088" title="Show "latest" run results via fixed URL (dashboard like feature)" class="issue-link" data-issue-key="JENKINS-41088">JENKINS-41088</a>. We are going to build a great page for a Pipeline that shows you the latest visualization of your Pipeline which will be perfect for big screens!</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/kzantow.png" title="kzantow's avatar" /> kzantow:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=285392&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-285392">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>I still don't think this is the same. Someone may be interested in looking at ONE EXACT pipeline execution.</p>
</pre>
</details>

</ul>
</details>

<p>I still don't think this is the same. Someone may be interested in looking at ONE EXACT pipeline execution.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/jamesdumay.png" title="jamesdumay's avatar" /> jamesdumay:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=285397&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-285397">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kzantow" class="user-hover" rel="kzantow">kzantow</a> <a href="https://issues.jenkins.io/browse/JENKINS-41088" title="Show &quot;latest&quot; run results via fixed URL (dashboard like feature)" class="issue-link" data-issue-key="JENKINS-41088">JENKINS-41088</a> is exactly the same as this feature request however there may be some details this ticket contributes that are not currently defined in it.</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kzantow" class="user-hover" rel="kzantow">kzantow</a> <a href="https://issues.jenkins.io/browse/JENKINS-41088" title="Show "latest" run results via fixed URL (dashboard like feature)" class="issue-link" data-issue-key="JENKINS-41088">JENKINS-41088</a> is exactly the same as this feature request however there may be some details this ticket contributes that are not currently defined in it.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>gusgonnet72:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=286028&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-286028">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>alright, thank you guys for the explanations.
should I close this ticket?</p>
</pre>
</details>

</ul>
</details>

<p>alright, thank you guys for the explanations.<br/>
should I close this ticket?</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/jamesdumay.png" title="jamesdumay's avatar" /> jamesdumay:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=286097&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-286097">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gusgonnet72" class="user-hover" rel="gusgonnet72">gusgonnet72</a> ill do that now.</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gusgonnet72" class="user-hover" rel="gusgonnet72">gusgonnet72</a> ill do that now.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>jgamble:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=320624&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-320624">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jamesdumay" class="user-hover" rel="jamesdumay">jamesdumay</a> Whilst I can agree that the ticket (<a href="https://issues.jenkins.io/browse/JENKINS-41088" title="Show &quot;latest&quot; run results via fixed URL (dashboard like feature)" class="issue-link" data-issue-key="JENKINS-41088">JENKINS-41088</a>) to create a dashboard view may address <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gusgonnet72" class="user-hover" rel="gusgonnet72">gusgonnet72</a>'s original ask, I don't think this ticket should be closed. It's still a completely valid user story to disable the auto-scroll functionality when viewing a pipeline as it can be jarring for users who don't typically consume log data first, but instead want to use the overview to drill down into failure areas they care about.</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jamesdumay" class="user-hover" rel="jamesdumay">jamesdumay</a> Whilst I can agree that the ticket (<a href="https://issues.jenkins.io/browse/JENKINS-41088" title="Show "latest" run results via fixed URL (dashboard like feature)" class="issue-link" data-issue-key="JENKINS-41088">JENKINS-41088</a>) to create a dashboard view may address <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gusgonnet72" class="user-hover" rel="gusgonnet72">gusgonnet72</a>'s original ask, I don't think this ticket should be closed. It's still a completely valid user story to disable the auto-scroll functionality when viewing a pipeline as it can be jarring for users who don't typically consume log data first, but instead want to use the overview to drill down into failure areas they care about.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/jamesdumay.png" title="jamesdumay's avatar" /> jamesdumay:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=320981&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-320981">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jgamble" class="user-hover" rel="jgamble">jgamble</a> the behaviour that you are describing is what the default is on master (1.4 release) right now. </p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jgamble" class="user-hover" rel="jgamble">jgamble</a> the behaviour that you are describing is what the default is on master (1.4 release) right now. </p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>kordzik:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=328432&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-328432">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jamesdumay" class="user-hover" rel="jamesdumay">jamesdumay</a> hello <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/smile.png" height="16" width="16" align="absmiddle" alt="" border="0"/> </p>

<p>We are on Blue Ocean 1.4.1, on a red build I still get the auto-scrolled to the stage and step that has failed. I'd argue that the behaviour by default should be the same, just land the user on the pipeline overview (top of the page) and let them scroll to the failed step(s) themselves. </p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jamesdumay" class="user-hover" rel="jamesdumay">jamesdumay</a> hello <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/smile.png" height="16" width="16" align="absmiddle" alt="" border="0"/> </p>

<p>We are on Blue Ocean 1.4.1, on a red build I still get the auto-scrolled to the stage and step that has failed. I'd argue that the behaviour by default should be the same, just land the user on the pipeline overview (top of the page) and let them scroll to the failed step(s) themselves. </p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/jamesdumay.png" title="jamesdumay's avatar" /> jamesdumay:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=329330&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-329330">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kordzik" class="user-hover" rel="kordzik">kordzik</a> yes, it automatically scrolls there on a failed run by design. I'd need to get some more feedback from users before we went ahead and changed this behaviour. </p>

<p>Also, hello - great to see that you're a Blue Ocean user now <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/smile.png" height="16" width="16" align="absmiddle" alt="" border="0"/></p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kordzik" class="user-hover" rel="kordzik">kordzik</a> yes, it automatically scrolls there on a failed run by design. I'd need to get some more feedback from users before we went ahead and changed this behaviour. </p>

<p>Also, hello - great to see that you're a Blue Ocean user now <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/smile.png" height="16" width="16" align="absmiddle" alt="" border="0"/></p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>fkykko:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=332068&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-332068">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jamesdumay" class="user-hover" rel="jamesdumay">jamesdumay</a> I would also support a change to where the top of the page is the default. I typically find myself scrolling up even in a failed build, just to see which part of the build that failed and then scrolling down to get the step log. </p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jamesdumay" class="user-hover" rel="jamesdumay">jamesdumay</a> I would also support a change to where the top of the page is the default. I typically find myself scrolling up even in a failed build, just to see which part of the build that failed and then scrolling down to get the step log. </p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>stevedonie:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=339834&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-339834">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jamesdumay" class="user-hover" rel="jamesdumay">jamesdumay</a> please consider this as my vote as a user (as well as my whole team) to not automatically scroll.</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jamesdumay" class="user-hover" rel="jamesdumay">jamesdumay</a> please consider this as my vote as a user (as well as my whole team) to not automatically scroll.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>jsnrth:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=352720&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-352720">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jamesdumay" class="user-hover" rel="jamesdumay">jamesdumay</a> ditto, my team and I would like to see this auto-scroll behavior disabled.</p>

<p>We find ourselves scrolling back to the top nearly every time we navigate to a failed build from links on Github checks. Further, while the page is loading, the top with the pipeline and menu is shown – it only auto-scrolls to the bottom once the JS or whatever kicks in and loads the console output. This makes the UX disruptive as it jumps us away from where we were getting situated on the page.</p>

<p>We would much prefer if the page did not assume we wanted to be at the bottom of the failed stage's console output, and instead allowed us to navigate where we want to go.</p>

<p>I see that this issue is closed as a duplicate of <a href="https://issues.jenkins.io/browse/JENKINS-41088" title="Show &quot;latest&quot; run results via fixed URL (dashboard like feature)" class="issue-link" data-issue-key="JENKINS-41088">JENKINS-41088</a>, but I don't think that is quite the same request (per <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jgamble" class="user-hover" rel="jgamble">jgamble</a>'s comment about a year ago).</p>

<p>We are running Blue Ocean 1.9.0.</p>

<p>Thanks, -j</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jamesdumay" class="user-hover" rel="jamesdumay">jamesdumay</a> ditto, my team and I would like to see this auto-scroll behavior disabled.</p>

<p>We find ourselves scrolling back to the top nearly every time we navigate to a failed build from links on Github checks. Further, while the page is loading, the top with the pipeline and menu is shown – it only auto-scrolls to the bottom once the JS or whatever kicks in and loads the console output. This makes the UX disruptive as it jumps us away from where we were getting situated on the page.</p>

<p>We would much prefer if the page did not assume we wanted to be at the bottom of the failed stage's console output, and instead allowed us to navigate where we want to go.</p>

<p>I see that this issue is closed as a duplicate of <a href="https://issues.jenkins.io/browse/JENKINS-41088" title="Show "latest" run results via fixed URL (dashboard like feature)" class="issue-link" data-issue-key="JENKINS-41088">JENKINS-41088</a>, but I don't think that is quite the same request (per <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jgamble" class="user-hover" rel="jgamble">jgamble</a>'s comment about a year ago).</p>

<p>We are running Blue Ocean 1.9.0.</p>

<p>Thanks, -j</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>cetex:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=353332&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-353332">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>This is definitely not resolved, or handled by the referenced issue.</p>

<p>Some builds we run ends up launching 15 parallel tasks (building AMI's for AWS environment), and i want to look at one of these.</p>

<p>The pipeline view's javascript makes it very very hard to select which one of those parallel tasks I want to watch as it's scrolling down to the bottom every few seconds, so i basically have to wait until everything is done (half an hour) before i can even select the right build and see what happened IN THE BEGINNING of this build to figure out what's wrong.</p>

<p>Next, if i manage to pick the correct parallel build it somehow still manages to switch the console output to the output of another build if that one's finished before the one i'm currently watching..</p>

<p>To be honest, this autoscrolling behavior is useless for most of our scenarios as we run a lot of stuff in parallel.</p>

<p>It might be ok to autoscroll-down if the user is already on the bottom of the page and you add more text, but don't mess with where my scrollbar is on the page otherwise. If i want to place it on the middle of the page to investigate something, let me do that. Currently the behaviour prohibits this as it's impossible to do anything but watch the last few lines as long as a build with a lot of text output is running.</p>

<p>Either disable autoscrolling completely or only make it apply whenever a user has scrolled down to bottom.</p>
</pre>
</details>

</ul>
</details>

<p>This is definitely not resolved, or handled by the referenced issue.</p>

<p>Some builds we run ends up launching 15 parallel tasks (building AMI's for AWS environment), and i want to look at one of these.</p>

<p>The pipeline view's javascript makes it very very hard to select which one of those parallel tasks I want to watch as it's scrolling down to the bottom every few seconds, so i basically have to wait until everything is done (half an hour) before i can even select the right build and see what happened IN THE BEGINNING of this build to figure out what's wrong.</p>

<p>Next, if i manage to pick the correct parallel build it somehow still manages to switch the console output to the output of another build if that one's finished before the one i'm currently watching..</p>

<p>To be honest, this autoscrolling behavior is useless for most of our scenarios as we run a lot of stuff in parallel.</p>

<p>It might be ok to autoscroll-down if the user is already on the bottom of the page and you add more text, but don't mess with where my scrollbar is on the page otherwise. If i want to place it on the middle of the page to investigate something, let me do that. Currently the behaviour prohibits this as it's impossible to do anything but watch the last few lines as long as a build with a lot of text output is running.</p>

<p>Either disable autoscrolling completely or only make it apply whenever a user has scrolled down to bottom.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/kzantow.png" title="kzantow's avatar" /> kzantow:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=353334&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-353334">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>I'd also note: <a href="https://issues.jenkins.io/browse/JENKINS-41088" title="Show &quot;latest&quot; run results via fixed URL (dashboard like feature)" class="issue-link" data-issue-key="JENKINS-41088">JENKINS-41088</a> is a slightly different ask, but could somewhat be accommodated by implementing this. I'll add that after using the tool for a while, I also find myself less interested in following along a particular stage than seeing the graph to click around. At one point, long ago, I did some work to "fix" this behavior somewhat, but it didn't disable following by default. Perhaps I can resurrect the pertinent parts of that if I get some time. Or perhaps a simple button to follow/not follow that's saved in browser storage or something similar would suffice; not sure the right UX for it &#8211; suggestions welcome.</p>
</pre>
</details>

</ul>
</details>

<p>I'd also note: <a href="https://issues.jenkins.io/browse/JENKINS-41088" title="Show "latest" run results via fixed URL (dashboard like feature)" class="issue-link" data-issue-key="JENKINS-41088">JENKINS-41088</a> is a slightly different ask, but could somewhat be accommodated by implementing this. I'll add that after using the tool for a while, I also find myself less interested in following along a particular stage than seeing the graph to click around. At one point, long ago, I did some work to "fix" this behavior somewhat, but it didn't disable following by default. Perhaps I can resurrect the pertinent parts of that if I get some time. Or perhaps a simple button to follow/not follow that's saved in browser storage or something similar would suffice; not sure the right UX for it &#8211; suggestions welcome.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>timethos:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=371126&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-371126">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Do we just have to kill this line?</p>

<p><a href="https://github.com/jenkinsci/blueocean-plugin/blob/b02273dc2a6aee295ac8bd7dbab8eb3a8516a60e/blueocean-dashboard/src/main/js/components/karaoke/components/LogConsole.jsx#L156" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/blueocean-plugin/blob/b02273dc2a6aee295ac8bd7dbab8eb3a8516a60e/blueocean-dashboard/src/main/js/components/karaoke/components/LogConsole.jsx#L156</a></p>

<p> </p>

<p>This scroll bottom thing hurts more than it helps. It's easy enough for a user to hit fn + right arrow, or the end key...</p>
</pre>
</details>

</ul>
</details>

<p>Do we just have to kill this line?</p>

<p><a href="https://github.com/jenkinsci/blueocean-plugin/blob/b02273dc2a6aee295ac8bd7dbab8eb3a8516a60e/blueocean-dashboard/src/main/js/components/karaoke/components/LogConsole.jsx#L156" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/blueocean-plugin/blob/b02273dc2a6aee295ac8bd7dbab8eb3a8516a60e/blueocean-dashboard/src/main/js/components/karaoke/components/LogConsole.jsx#L156</a></p>

<p> </p>

<p>This scroll bottom thing hurts more than it helps. It's easy enough for a user to hit fn + right arrow, or the end key...</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/mspiller.svg" title="mspiller's avatar" /> mspiller:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-41488?focusedId=422747&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-422747">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Any chance to get auto scrolling disabled? It is making blueocean very hard to use.</p>

<p>Every time there is an output update from a build step the page is auto scrolled down. I understand this can be perhaps useful the first (once) time but not every time.</p>

<p>When you have parallel builds. Click on one build that outputs a lot and then click Home on the keyboard or Page Up or any other key.</p>

<p>After receiving new output page is scrolled to the bottom. And you can repeat this over and over again.</p>

<p>The scrolling behaviour stops if I scroll the mouse wheel but any keypress (page up, page down, home, end, up, down) is keeping messing up the scrolling position.</p>
</pre>
</details>

</ul>
</details>

<p>Any chance to get auto scrolling disabled? It is making blueocean very hard to use.</p>

<p>Every time there is an output update from a build step the page is auto scrolled down. I understand this can be perhaps useful the first (once) time but not every time.</p>

<p>When you have parallel builds. Click on one build that outputs a lot and then click Home on the keyboard or Page Up or any other key.</p>

<p>After receiving new output page is scrolled to the bottom. And you can repeat this over and over again.</p>

<p>The scrolling behaviour stops if I scroll the mouse wheel but any keypress (page up, page down, home, end, up, down) is keeping messing up the scrolling position.</p>
